x-pack/docs/en/security/authentication/configuring-native-realm.asciidoc

```diff
@@ -21,13 +28,9 @@ sets the `order` to zero so the realm is checked first:
 
 [source, yaml]
 ------------------------------------------------------------
-xpack:
-  security:
-    authc:
-      realms:
-        native:
-          native1:
-            order: 0
+xpack.security.authc.realms:
+  native.native1:
+      order: 0
```

Comment:
Nit: I don't particularly like the existing style because it is a bit too hierarchical and verbose. I think the more compact style looks better. But I have a few comments:
* Should we prefer it in future more consistently in other places? We don't need to touch anything else in this PR. But it would be good to know the future direction.
* Should we place `native.native1` on the same line as `xpack.security.authc.realms`? I think it is even more concise and direct to the context?
* The `order` field has a different amount of indentation.

Reply:
I think we should prefer it in future and make it more consistent in other places. I'll place `native.native1` onto the same line as `xpack.security.authc.realms` and fix the indentation issue with `order`. Thanks for catching that!